exec/vector/src/main/java/org/apache/drill/exec/record/MaterializedField.java

```diff
@@ -38,6 +41,8 @@
  */
 
 public class MaterializedField {
+  public final static MaterializedField OFFSETS_FIELD = new MaterializedField(ValueVector.OFFSETS_VECTOR_NAME, Types.required(MinorType.UINT4), 0);
```

Comment:
OK to put in a value vector, but not here. The idea of an offsets field is a vector concept, not a metadata concept.

Reply:
I disagree. I don't see any problem with static `OFFSET_FIELD` belonging to `MaterializedField`.